src/sentry/utils/sdk_crashes/event_stripper.py

```diff
@@ -0,0 +1,130 @@
+from enum import Enum, auto
+from typing import Any, Dict, Mapping, Optional, Sequence
+
+from sentry.eventstore.models import Event
+from sentry.utils.safe import get_path
+from sentry.utils.sdk_crashes.sdk_crash_detector import SDKCrashDetector
+
+
+class Allow(Enum):
```

Comment:
Code of this file was reviewed in https://github.com/getsentry/sentry/pull/50400.